libs/design/src/atoms/form/radio/radio.component.ts

```diff
@@ -65,12 +65,12 @@ export class DaffRadioComponent implements OnInit {
    * Used for aria-label. Default to name if user does not input a label.
    */
   //tslint:disable-next-line:no-input-rename
-  @Input('aria-label') label = name;
+  @Input('aria-label') label: string = name;
   /**
    * Used for aria-labelledby. 
    */
   //tslint:disable-next-line:no-input-rename
-  @Input('aria-labelledby') labelledby;
+  @Input('aria-labelledby') labelledby: string;
```

Comment:
These errors were caused by the typescript upgrade: https://github.com/microsoft/TypeScript/issues/31663

I guess it requires stronger typing in some instances